prow/config.yaml

```diff
@@ -55,6 +55,19 @@ prowjob_namespace: default
 pod_namespace: test-pods
 log_level: debug
 
+slack_reporter:
+  job_types_to_report:
+  - presubmit
```

Comment:
I don't think we want to report presubmit failures. That'd be a lot of noise.

Reply:
presubmits might be okay if it messaged the person who authored the PR, but that isn't how it works.

Reply:
Removed.